.pre-commit-config.yaml

```diff
@@ -6,26 +6,33 @@
     language: system
     files: >-  # begin multiline string
       (?x)^(   # (?x) -> multiline regex, ^ -> beginning of file path
+        .ci/test_plugin_testcase.py|
+        .ci/test_setup.py|
+        aiida/backends/tests/cmdline/commands/test_calculation.py|
+        aiida/backends/tests/cmdline/commands/test_export.py|
+        aiida/backends/tests/cmdline/commands/test_rehash.py|
+        aiida/backends/tests/cmdline/commands/test_work.py|
+        aiida/backends/tests/verdi_commands.py|
         aiida/control/.*.py|        # must match the whole path, therefore
         aiida/cmdline/tests/.*.py|  # the .*.py (.* matches everything)
+        aiida/cmdline/commands/calculation.py|
         aiida/cmdline/commands/daemon.py|
         aiida/cmdline/commands/node.py|
+        aiida/cmdline/commands/rehash.py|
+        aiida/cmdline/commands/restapi.py|
         aiida/cmdline/params/.*.py|
         aiida/cmdline/optiontypes/.*.py|
         aiida/cmdline/utils/multi_line_input.py|
         aiida/cmdline/utils/test_multiline.py|
         aiida/cmdline/utils/templates.py|
         aiida/cmdline/utils/echo.py|
-        docs/update_req_for_rtd.py| # a|b -> match a OR b
-        .ci/test_setup.py|
-        aiida/transport/.*.py|
         aiida/scheduler/.*.py|
-        .ci/test_plugin_testcase.py|
-        aiida/backends/tests/verdi_commands.py|
+        aiida/transport/.*.py|
         aiida/orm/data/cif.py|
         aiida/restapi/resources.py|
         aiida/restapi/translator/data/cif.py|
-        aiida/utils/fixtures.py
+        aiida/utils/fixtures.py|
+        docs/update_req_for_rtd.py| # a|b -> match a OR b
```

Comment:
I'm not sure it should finish with a vertical pipe `|`

Reply:
Only for that line, or for all of them?

Reply:
If I understand the syntax, the | is an "OR" so it should be everywhere except the last line (btw I'm still not convinced that the syntax is perfectly ok - e.g. the dot probably matches any character?)

Reply:
Ah, yes I did not recognize that you meant the last line. I think you are right and I removed it. The dot should indeed match any character. I tried removing it before the wildcard and then it does not match anything. It really seems to work like a regex pattern then